readme.md

```diff
@@ -12,3 +12,33 @@ There are a couple of ways to setup the local development environment:
 
 - [Developing inside Docker](https://github.com/AutoFixture/AutoFixture.github.io/wiki/Docker-Environment-Setup)
 - [Developing locally](https://github.com/AutoFixture/AutoFixture.github.io/wiki/Local-Environment-Setup)
+
+## Roadmap 🚧
+
+> :warning: There is currently an ongoing effort to document AutoFixture.
+> Any contributions to the documentation are welcome.
+
+- [ ] Overview
+- [ ] Quick Start guide
+- [ ] Fixture creation
+- [ ] Fixture customization
+  - [ ] Customize
+  - [ ] Build
+- [ ] Freeze and Inject features
+- [ ] Testing frameworks integration
+  - [ ] xUnit
+  - [ ] NUnit
+- [ ] Mocking framework integration
+  - [ ] Moq
+  - [ ] NSubstitute
+  - [ ] FakeItEasy
+  - [ ] Foq
+  - [ ] RhinoMock
+- [ ] Idioms
+  - [ ] Assertions
+  - [ ] FsCheck
+- [ ] Internal library design and philosophy
```

Comment:
Add `Behaviors` here as well. They are a sort of customizations, but a bit different and complicated.

Reply:
done